### ISSUE TITLE
Code for Tabs is not used anymore

### ISSUE THREAD

**Issue by christinem** [Courseography/courseography#455]
The file setup_tabs.js doesn't seem to be needed anymore, as well as there is some lines of code in setup.js related to that and the old FCE count that are not needed anymore either.

**Comment by Ian-Stewart-Binks** [comment]
@christinem Will you not be using any of the code in that directory for focuses/fce counting? I'm going to completely remove this directory- is there anything that you want me to save?

**Comment by christinem** [comment]
@Ian-Stewart-Binks they are both used actually (focuses.js and fce_count.js), but not currently in the master branch, but in my sidebar branch. Perhaps you can remove them from master, and I can move the files over to graphs/sidebar in the sidebar branch, so that's where they'll end up when sidebar eventually gets merged into master.

**Comment by Ian-Stewart-Binks** [comment]
You're using `updateClickedCourses` right? I will need to find a home for that function before we can remove the tab code completely, as it's a function that's used in two places outside of the tab code.

**Comment by christinem** [comment]
Yes, I am using that function as well.

**Comment by david-yz-liu** [comment]
@christinem We're going to get rid of the graph/tabs js directory. Please move whatever functions you need into one of your new files.

**Comment by christinem** [comment]
@david-yz-liu Will do!
